src/transactions/utils/getTransactionBytes.js

```diff
@@ -150,43 +150,48 @@ export const checkTransaction = transaction => {
 };
 
 const getTransactionBytes = transaction => {
+	const {
+		type,
+		timestamp,
+		requesterPublicKey,
+		senderPublicKey,
+		recipientId,
+		amount,
+		signature,
+		signSignature,
+	} = transaction;
```

Comment:
Do we prefer this syntax?

Reply:
I like it, but maybe it makes more sense for `checkTransaction(transaction)` to come before the destructuring?